.github/workflows/lint.yml

```diff
@@ -1,13 +1,14 @@
 name: Lint
 
 on:
-  # Trigger the workflow on push or pull request,
-  # but only for the main branch
   push:
     branches: [main]
   pull_request:
-    branches:
-      - "*"
+    paths-ignore:
+      - 'docs/**'
+      - 'docusaurus/**'
+  schedule:
+    - cron: '0 7 * * *'
```

Comment:
Who gets notified if this fails since it's not tied to a PR?

Reply:
Yes. We got notified whenever an action fails, no matter how it is triggered.

Reply:
Who is we in this case? Anyone watching the repo? Contributors?

Reply:
Who is watching the repo:

https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/configuring-notifications#github-actions-notification-options